zookeeper/src/main/java/com/linecorp/armeria/server/zookeeper/listener/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/**
+ * A ZooKeeper Server Listener.When you add this listener, server will be automatically registered
+ * into the ZooKeeper.
+ *
+ * {@link com.linecorp.armeria.server.zookeeper.listener.ZooKeeperListener} can register server to a ZooKeeper
+ * cluster.The registered ZooKeeper node is EPHEMERAL, so when server stops, the underlying ZooKeeper session
+ * will be closed. The node will be automatically removed by ZooKeeper cluster.
+ */
+package com.linecorp.armeria.server.zookeeper.listener;
```

Comment:
We could probably merge this package into `com.linecorp.armeria.server.zookeeper`?